<?xml version="1.0"?>
<launch>
  <arg name="voxel_size" default="0.2" />
  <arg name="voxels_per_side" default="16" />
  <arg name="frame_id" default="world" />

  <node name="voxblox_skeleton_planner" pkg="voxblox_skeleton_planner" type="skeleton_global_planner_node" args="-minloglevel=2" output="screen" clear_params="true">
    <remap from="~voxblox" to="/voxblox_skeletonizer/skeleton_and_graph_file" />
  
    <param name="visualize" value="true" />
    <param name="robot_radius" value="0.5" />
    <param name="tsdf_voxel_size" value="$(arg voxel_size)" />
    <param name="tsdf_voxels_per_side" value="$(arg voxels_per_side)" />
    <param name="update_mesh_every_n_sec" value="0.0" />
    <param name="color_mode" value="normals" />
    <param name="frame_id" value="$(arg frame_id)" />
    <param name="world_frame" value="$(arg frame_id)" />
    <param name="trust_approx_solution" value="true" />
    <param name="publish_traversable" value="true" />
    <param name="split_at_collisions" value="true" />
    <param name="optimize_time" value="true" />
    <param name="verbose" value="false" />
  </node>

</launch>
